docs/config.md

```diff
@@ -293,6 +293,34 @@ This option defines the request timeout value for sending requests retrieving a
 
 By default, this value is set to 4000 milliseconds.
 
+
+### Option "cf.request.backoff.initial" (optional)
+
+When Promregator is sending metadata requests to the Cloud Foundry platform and receives an error, it will automatically retry the request up to two further attempts. However, it will not do so immediately on receiving the error (as this could lead to flooding of an already failing server). Instead it will retry only after waiting for a short amount of time - the backoff interval. For each repeated failure, the backoff interval is doubled, leading to an exponential increase of this interval the more failures are reported (giving the server more time to recover). Furthermore, a random duration is added to prevent peaks on retry attempts, which also can be hard for the server to survive.
+
+This option defines the initial duration of the backoff interval.
+
+The unit of this option is in milliseconds. By default, this value is set to 500 milliseconds.
+
+### Option "cf.request.backoff.max" (optional)
+
+When Promregator is sending metadata requests to the Cloud Foundry platform and receives an error, it will automatically retry the request up to two further attempts. However, it will not do so immediately on receiving the error (as this could lead to flooding of an already failing server). Instead it will retry only after waiting for a short amount of time - the backoff interval. For each repeated failure, the backoff interval is doubled, leading to an exponential increase of this interval the more failures are reported (giving the server more time to recover). Furthermore, a random duration is added to prevent peaks on retry attempts, which also can be hard for the server to survive.
+
+This option defines the maximal duration of the backoff interval (including the random duration) - the backoff interval will never grow above this value.
+
+The unit of this option is in milliseconds. By default, this value is set to 1200 milliseconds.
+
+### Option "cf.request.rateLimit" (optional)
+
+Promregator is able to send large amounts of requests to the Cloud Foundry platform. Due to its design, in large environments it is even possible that too many requests are sent in a too short time. In this case, the Cloud Foundry platform may [take protective measures for self-protection](https://docs.cloudfoundry.org/running/rate-limit-cloud-controller-api.html). Due to this, corresponding requests will fail, because they will be completed with various types of error messages.
+
+This option defines the number of requests per second that Promregator is allowed to send for fetching metadata from the Cloud Foundry platform. Requests which exceed this threshold are queued and will be processed only after enough "capacity" is available again.
+
+By default, this value is set to 0 (zero), which means that rate limiting is disabled.
```

Comment:
I think the `POSITIVE_INFIINITY` option is fine. My reasoning for proposing a lower default would be to have Promregator try to be a good citizen and allow it to run in a 'pretty good' default mode so it could startup and grab the values.

I'm hoping to create a Grafana standardized dashboard soon that could include a graph of current the throughput so a user could refer to that help them see if their cache/requests need to be adjusted.